examples/azure/postgres/build.sh

```diff
@@ -31,12 +32,28 @@ server <= azure_postgres_set_max_size($server, $size)
 server <= azure_postgres_set_version($server, $version)
 
 if $ssl == "false" {
-        server <= azure_postgres_disable_ssl($server)
+	server <= azure_postgres_disable_ssl($server)
 }
 
 azure_postgres_server_create($server)
 
 echo "created Postgres Server"
+echo "creating Postgres Server Firewall rule"
+
+publicip, status <= curl ipinfo.io/ip
+
+if $status != "0" {
+	publicip = "8.8.8.8"
```

Comment:
Ok, the `curl` was a stupid idea but I have use some ip for the parameter. So I figured "why not?". But looking at it now I realize it can confuse people. What do you suggest?